docs/install.rst

```diff
@@ -106,40 +110,37 @@ manually setup ``icalendar`` like this.
 
 Try it out:
 
-.. code-block:: python
+.. code-block:: pycon
 
     Python 3.9.5 (default, Nov 23 2021, 15:27:38)
     Type "help", "copyright", "credits" or "license" for more information.
     >>> import icalendar
     >>> icalendar.__version__
     '5.0.13'
 
-Building the documentation
---------------------------
+Build the documentation
+-----------------------
+
+To build the documentation, follow these steps:
 
-To build the documentation follow these steps:
+.. code-block:: shell
 
-.. code-block:: bash
+    source .tox/py311/bin/activate
+    pip install -r requirements_docs.txt
+    cd docs
+    make html
 
-    $ source .tox/py39/bin/activate
-    $ pip install -r requirements_docs.txt
-    $ cd docs
-    $ make html
+You can now open the output from ``_build/html/index.html``.
 
-You can now open the output from ``_build/html/index.html``. To build the
-presentation-version use ``make presentation`` instead of ``make html``. You
-can open the presentation at ``presentation/index.html``.
+To build the documentation, view it in a web browser, and automatically reload changes while you edit documentation, use ``make livehtml`` instead of ``make html``.
```

Comment:
personally, for the easy copy&paste style, I would prefer those commands to do into a codeblock.

Reply:
```
.. code-block:: shell

    make livehtml 
```

Then, it is consistent in style and suggests that it is of equal value.

Reply:
Could you do that? I think it is fine to merge already.

Reply:
I could do one even better. Would you find this useful?

https://sphinx-copybutton.readthedocs.io/en/latest/index.html

I use it in Plone docs.

Reply:
I'll reformat as code block, and put the plug-in in a new issue and PR.

Reply:
Done in 8581d355d6250b27ca9d8708d7c206f66cee73e4